code/game/objects/structures/statues.dm

```diff
@@ -0,0 +1,376 @@
+				return
+			playsound(loc, 'sound/items/Ratchet.ogg', 100, 1)
+			user.visible_message("[user] is securing the [name]'s bolts...", \
+								 "<span class='notice'>You are securing the [name]'s bolts...</span>")
+			if(do_after(user, 40, target = src))
+				if(!loc || anchored)
+					return
+				user.visible_message("[user] has secured the [name]'s bolts.", \
+									 "<span class='notice'>You have secured the [name]'s bolts.</span>")
+				anchored = 1
+
+	else if(istype(W, /obj/item/weapon/gun/energy/plasmacutter))
+		playsound(src, 'sound/items/Welder.ogg', 100, 1)
+		user.visible_message("[user] is slicing apart the [name]...", \
+							 "<span class='notice'>You are slicing apart the [name]...</span>")
+		if(do_after(user, 40, target = src))
+			if(!loc)
+				return
+			user.visible_message("[user] slices apart the [name].", \
+								 "<span class='notice'>You slice apart the [name].</span>")
+			Dismantle(TRUE)
+
+	else if(istype(W, /obj/item/weapon/pickaxe/drill/jackhammer))
+		var/obj/item/weapon/pickaxe/drill/jackhammer/D = W
+		if(!loc)
+			return
+		user.visible_message("[user] destroys the [name]!", \
+							 "<span class='notice'>You destroy the [name].</span>")
+		D.playDigSound()
+		qdel(src)
+
+	else if(iswelder(W) && !anchored)
+		playsound(loc, 'sound/items/Welder.ogg', 40, 1)
+		user.visible_message("[user] is slicing apart the [name].", \
+							 "<span class='notice'>You are slicing apart the [name]...</span>")
+		if(do_after(user, 40, target = src))
+			if(!loc)
+				return
+			playsound(loc, 'sound/items/Welder2.ogg', 50, 1)
+			user.visible_message("[user] slices apart the [name].", \
+								 "<span class='notice'>You slice apart the [name]!</span>")
+			Dismantle(TRUE)
+	else
+		hardness -= W.force/100
+		..()
+		CheckHardness()
+		return ..()
+
+/obj/structure/statue/attack_hand(mob/living/user)
+	user.changeNext_move(CLICK_CD_MELEE)
+	add_fingerprint(user)
+	user.visible_message("[user] rubs some dust off from the [name]'s surface.", \
+						 "<span class='notice'>You rub some dust off from the [name]'s surface.</span>")
+
+/obj/structure/statue/CanAtmosPass()
+	return !density
+
+/obj/structure/statue/bullet_act(obj/item/projectile/Proj)
+	hardness -= Proj.damage
+	..()
+	CheckHardness()
+	return
+
+/obj/structure/statue/proc/CheckHardness()
+	if(hardness <= 0)
+		Dismantle(TRUE)
+
+/obj/structure/statue/proc/Dismantle(disassembled = TRUE)
+	if(material_drop_type)
+		var/drop_amt = oreAmount
+		if(!disassembled)
+			drop_amt -= 2
+		if(drop_amt > 0)
+			new material_drop_type(get_turf(src), drop_amt)
+	qdel(src)
+
+/obj/structure/statue/ex_act(severity = 1)
+	switch(severity)
+		if(1)
+			Dismantle(TRUE)
+		if(2)
+			if(prob(20))
+				Dismantle(TRUE)
+			else
+				hardness--
+				CheckHardness()
+		if(3)
+			hardness -= 0.1
+			CheckHardness()
+	return
+
+/obj/structure/statue/uranium
+	hardness = 3
+	luminosity = 2
```

Comment:
`light_range`

Never adjust luminosity, lest you cause the lighting apocalypse.